target/linux/ath79/image/Makefile

```diff
@@ -66,6 +74,28 @@ define Device/Default
 	append-rootfs | pad-rootfs | append-metadata | check-size $$$$(IMAGE_SIZE)
 endef
 
+define Device/seama
+$(Device/Default)
+  LOADER_TYPE := bin
+  KERNEL := kernel-bin | append-dtb | relocate-kernel | lzma
+  KERNEL_INITRAMFS := kernel-bin | append-dtb | relocate-kernel | lzma | seama
+  KERNEL_INITRAMFS_SUFFIX = $$(KERNEL_SUFFIX).seama
```

Comment:
yes, can be done. But I don't see much benefit of doing so as long as we are only able to create a single initramfs image.

Reply:
This is copied from the ar71xx target. I also think this is useless. I'll remove these lines.